plugins/repository-gcs/README.md

```diff
@@ -0,0 +1,37 @@
+Google Cloud Storage Plugin
```

Comment:
Not sure what to do with this file. Just leave it as it is or maybe convert it to a rst ?

Reply:
Remove it. Anything relevant mentioned here should be integrated as package/class level docs or go into the user documentation.